### PR TITLE
Merging the feature/ml branch into master

### PR DESCRIPTION
**Pull request by yebrahim**
Branch has been rebased to latest master. Ran the Iris sample preprocessing and training (locally) and it looks fine.

### CODE REVIEW

File: datalab/mlalpha/_dataset.py [hunk 1]
```diff
@@ -0,0 +1,403 @@
+import google.cloud.ml.features as features
```
Comment (ojarjur) [inline datalab/mlalpha/_dataset.py]: This file is missing the copyright header.

File: datalab/mlalpha/_cloud_predictor.py~48ec5ba7df5b895901affdea7c5ca5d14e688a4f [hunk 2]
```diff
@@ -0,0 +1,94 @@
+# Copyright 2016 Google Inc. All rights reserved.
```
Comment (qimingj) [inline datalab/mlalpha/_cloud_predictor.py~48ec5ba7df5b895901affdea7c5ca5d14e688a4f]: This file looks like a backup file created by some editor, and shouldn't be included.

File: datalab/mlalpha/_cloud_runner.py~48ec5ba7df5b895901affdea7c5ca5d14e688a4f [hunk 3]
```diff
@@ -0,0 +1,92 @@
+# Copyright 2016 Google Inc. All rights reserved.
```
Comment (qimingj) [inline datalab/mlalpha/_cloud_runner.py~48ec5ba7df5b895901affdea7c5ca5d14e688a4f]: Same for this file.

Comment (yebrahim) [inline datalab/mlalpha/_cloud_runner.py~48ec5ba7df5b895901affdea7c5ca5d14e688a4f]: fixed

**Pull request merged**